.devcontainer/postCreateCommand.sh

```diff
@@ -14,6 +14,3 @@ sudo apt-get install -y dialog
 
 # Install air to support live reloading of server on code changes
 go install github.com/air-verse/air@latest
-
-# Install golangci-lint to allow running of linting locally
-curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(go env GOPATH)/bin" v1.57.2
```

Comment:
Installing linter manually should not be needed as it's already included in the `golang:1.23.2-bookworm` base image.